Sources/StreamChatUI/UIConfig+ChannelList.swift

```diff
@@ -62,16 +62,7 @@ public extension _UIConfig {
         }
 
         /// The subviews that compose the `SwipeableView`.
-        public struct SwipeableItemSubviews {
-            /// The main content view which you should always use for embedding your cell content.
-            public var cellContentView: UIView.Type = UIView.self
-
-            /// The delete button.
-            public var deleteButton: UIButton.Type = UIButton.self
-
-            /// The more actions button.
-            public var moreActionsButton: UIButton.Type = UIButton.self
-
+        public struct SwipeableViewSubviews {
             /// The `UIStackView` that arranges buttons revealed by swipe gesture.
             public var actionButtonStack: UIStackView.Type = UIStackView.self
         }
```

Comment:
Hmmm, let's remove it. This is too specific. I don't think we need it in UIConfig

Reply:
Done!